src/ml/neural_net/mps_layer_helper.h

```diff
@@ -0,0 +1,41 @@
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h>
+#import <Metal/Metal.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.14))
+@interface TCMPSLayerHelper: NSObject
+
++ (MPSCNNFullyConnectedNode *) createFullyConnected:(MPSNNImageNode *)inputNode
+                               inputFeatureChannels:(NSUInteger)inputFeatureChannels
+                              outputFeatureChannels:(NSUInteger)outputFeatureChannels
+                                        inputHeight:(NSUInteger)inputHeight
+                                         inputWidth:(NSUInteger)inputWidth
+                                            weights:(float *)weights
+                                             biases:(float *)biases
+                                              label:(NSString *)label
+                                      updateWeights:(bool)updateWeights
```

Comment:
Nit: for Obj-C is more idiomatic to use BOOL

Reply:
updated